pylint/checkers/classes.py

```diff
@@ -416,6 +416,10 @@ def _has_same_layout_slots(slots, assigned_value):
               'Used whenever we can detect that an overridden method is useless, '
               'relying on super() delegation to do the same thing as another method '
               'from the MRO.'),
+    'W0236': ('Class defines only one of __eq__ and __ne__',
+              'both-equality-methods',
```

Comment:
message ids tends to be more descriptive about the error encountered.

Maybe use something along the lines of: `complementary-dunder-method-missing`?